modules/ephemeris/src/main/scala/gem/horizons/EphemerisContext.scala

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem
+package horizons
+
+import gem.enum.Site
+import gem.util.Timestamp
+
+import cats.implicits._
+import scala.math.Ordering.Implicits._
+
+
+/** Collection of information related to an ephemeris.  Used to determine
+  * whether an update is needed.
+  *
+  * @param key  unique horizons id of the object
+  * @param site information valid for the given site
+  * @param meta associated ephemeris metadata, if any
+  * @param rnge earliest and latest ephemeris element times, if any
+  * @param soln current horizons solution reference from JPL, if any
+  */
+final case class EphemerisContext(
```

Comment:
This was just pulled out of another class to the top level so make it easily available.